src/server.js

```diff
@@ -12,6 +12,7 @@ const helpers = require('./server.helpers');
 const v2 = require('./api/v2');
 const config = require('./config');
 const semver = require('semver');
+require('../scripts/init-new-db');
```

Comment:
too sneaky?

Reply:
Maybe a little bit? I can see this getting lost and people having to figure it out.

Should we create an `initDb()` function then do `const { initDb } = require('../scripts/init-new-db')` and explicitly call it?